include/oneapi/dpl/pstl/utils.h

```diff
@@ -287,6 +287,42 @@ class __transform_functor
     }
 };
 
+template <typename _UnaryOper, typename _UnaryPred>
+class __transform_if_unary_functor
+{
+    mutable _UnaryOper _M_oper;
```

Comment:
I see that in our code there are not so many usages of ```metable``` keywords for declare some class members.
For example, ```struct __brick_fill_n``` is implemented without this declaration.

May be make sense to avoid the usage of ```mutable``` keyword in this place too ?

Reply:
Please see my comments above https://github.com/oneapi-src/oneDPL/pull/276#discussion_r1192403770

Reply:
OK, thanks.